AK/Function.h

```diff
@@ -254,9 +276,11 @@ class Function<Out(In...)> {
         other.m_kind = FunctionKind::NullPointer;
     }
 
+    size_t m_size { 0 };
```

Comment:
Can't that think be a compile-time variable?

Reply:
I think it can't. `m_size` might change in runtime by `operator=` for example.

Reply:
Sure, I didn't think of that case, my bad!